how-to-guides/bubs-testnet.md

```diff
@@ -9,7 +9,7 @@ next:
 
 ![Bubs testnet](/img/Celestia_Bubs_Testnet.jpg)
 
-[Bubs Testnet](https://bubs-sepolia.hub.caldera.xyz/) is a the first
+[Bubs Testnet](https://bubstestnet.com/) is a the first
```

Comment:
It's not 100% clear to me if this testnet is still supported, and if this is the correct link to use. Please do lmk if this is correct _before_ we merge this :pray:

Reply:
this works as an archive link!